test/11_array.jl

```diff
@@ -522,6 +522,56 @@ for isordered in (false, true)
             @test x[1] === x.pool.valindex[3]
             @test x[2] === x.pool.valindex[1]
             @test levels(x) == ["c", "a", "b"]
+
+            a1 = 3:200
```

Comment:
Can you add a comment saying what you're testing here? It would be nice to have one general title about testing `vcat` (with two line breaks before), and comments for each test explaining what's the difference with the other ones you add.